.cirrus.yml

```diff
@@ -196,3 +196,25 @@ task:
     - rm /etc/ld.so.cache
     - ./ci/cirrus.sh
   << : *CAT_LOGS
+
+task:
+  name: "x86_64 (mingw32-w64): Windows (Debian stable, Wine)"
+  container:
+    dockerfile: ci/linux-debian.Dockerfile
+    cpu: 1
+    memory: 1G
+  env:
+    WINE_CMD: wine
+    HOST: x86_64-w64-mingw32
+    BUILD:
+    WITH_VALGRIND: no
+    ECDH: yes
+    RECOVERY: yes
+    EXPERIMENTAL: yes
+    SCHNORRSIG: yes
+    CTIMETEST: no
+    CFLAGS: -O2 -static -static-libgcc
```

Comment:
Do we need the static flags?  I guess it's just my lack of experience with mingw but maybe add a comment.

Also -O2 is our default, so we probably shouldn't set it explicitly here.